mbuild/compound.py

```diff
@@ -69,53 +69,65 @@ def load(filename, relative_to_module=None, compound=None, coords_only=False,
     compound : mb.Compound
 
     """
+    # First check if we are loading from an existing parmed or trajectory structure
+    if isinstance(filename_or_topology, pmd.Structure):
+        new_structure = Compound()
+        new_structure.from_parmed(filename_or_topology)
```

Comment:
What about the `coords_only` flag? In other sections of `mb.load`, we pass the `coords_only` flag when using mdtraj or parmed readers

Reply:
I don't think the `coords_only` flag applies to this case. Based on what I understand, the `coords_only` flag is used when we try to apply the coordinates (but not the atoms) from a mdtraj or parmed structure to an existing `Compound` instance, but here we try to create a new `Compound` instance. In other words, if the `coords_only` ever evaluated to `True`, we will have a `Number of atoms does not match` error.

Reply:
I am also thinking of the possibility that we want to use `mb.load()` in a more dynamic way, like using it to apply the `coords_only` to an existing `mb.Compound` instance, then it would be meaningful to have the `coords_only` flag in place. I will try to work in this direction tomorrow.

Reply:
@ahy3nz I added your suggestions into the code. Now we can do something like
 `compound = mb.load(file_or_top, compound=compound, coords_only=True)`
to update only the coordinates of the `compound` from exisiting `parmed` or `mdtraj` structure.